src/dparse/ast.d

```diff
@@ -2726,6 +2728,20 @@ final class SpecifiedFunctionBody : BaseNode
     mixin OpEquals;
 }
 
+///
+final class ShortenedFunctionBody : BaseNode
+{
+    override void accept(ASTVisitor visitor) const
+    {
+        mixin(visitIfNotNull!(expression));
+    }
+
+    /** */ Expression expression;
+    /** */ size_t startLocation;
+    /** */ size_t endLocation;
```

Comment:
startLocation & endLocation never actually assigned (and obsolete since the introduction of the tokens slice)